### DISCUSSION TITLE
likelihood eval mode on approximate GP

### DISCUSSION THREAD

**Discussion by wollschlager** [cornellius-gp/gpytorch#2055] [Q&A]
Hi, 
I have a question regarding the approximate GP: in the [example for stochastic regression](https://docs.gpytorch.ai/en/stable/examples/04_Variational_and_Approximate_GPs/SVGP_Regression_CUDA.html), why don't we need to wrap the prediction of the model with the likelihood during evaluation? It is done in the [example](https://docs.gpytorch.ai/en/stable/examples/01_Exact_GPs/Simple_GP_Regression.html) for the ExactGP?

And if that is no mistake, do we need to call likelihood.eval() even in the approximate GP case?

**Comment by gpleiss** [comment]
In the example that you link to, we are only evaluating the predictive mean, which is the same as the latent mean. Wrapping the model in the likelihood will only affects the variance (it returns the latent variance plus the observational variance, rather than just the latent variance).
